### PR TITLE
add ---document-issue-.md

### PR DESCRIPTION
**Pull request by TCChenlong**
### PR types

others
### PR changes

docs
### Describe

add document issue template

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/---document-issue-.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+#### Sample code is clear?
+For example: there is no sample code; the sample code is not instructable; the sample code does not work; the sample code format is problematic; the sample code has no comments;
+
+#### Chinese content and English content is consistent?
+For example: Chinese and English API descriptions are inconsistent; Chinese and English API parameters are inconsistent;
+
+#### Other
+For example: the document page cannot be opened; the document is missing; there is a dead link in document;
+
+**To Reproduce**
```
Comment (jzhang533) [inline .github/ISSUE_TEMPLATE/---document-issue-.md]: to reproduce这段不需要吧。

File: .github/ISSUE_TEMPLATE/---document-issue-.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+非常感谢您提交关于飞桨文档的Issue，我们会认真听取您的意见，并进行改进。
+
+建立issue时，为快速解决问题，请您根据情况给出如下信息：
+- 标题：请包含关键词“XXX文档问题”，例如“add 文档问题” 或 ”paddle.add 文档问题“
+- 文档版本信息：请提供有问题的文档的版本号，例如 develop，1.8，2.0RC；
+
+### 文档问题描述：
+
+#### API文档描述是否清晰？
+如：文档描述看不懂，不知道这个API该什么用；文档公式错误；
```
Comment (dingjiaweiww) [inline .github/ISSUE_TEMPLATE/---document-issue-.md]: 不知道这个API该“怎么”用，错字

File: .github/ISSUE_TEMPLATE/---document-issue-.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+#### 参数说明是否清晰
+如：没有解释清楚这个参数的作用以及使用场景；参数说明有误；
```
Comment (dingjiaweiww) [inline .github/ISSUE_TEMPLATE/---document-issue-.md]: 参数未解释清楚，包括用法、使用场景、默认值等

File: .github/ISSUE_TEMPLATE/---document-issue-.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+#### 返回/形状说明是否清晰
+如：返回说明错误；形状描述不清楚；
```
Comment (dingjiaweiww) [inline .github/ISSUE_TEMPLATE/---document-issue-.md]: api返回值、数据的形状描述错误、不清楚

File: .github/ISSUE_TEMPLATE/---document-issue-.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+如：没有示例代码；示例代码没有可指导性；示例代码跑不通；示例代码格式有问题；示例代码没有注释；
+
+#### 中英文内容是否一致？
+如：中英文API描述不一致；中英文API参数不一致；
+
+#### 其他
+如：文档页面打不开；文档缺失；文档中有死链；
+
+
+Thank you for submitting a PaddlePaddle documentation issue. We will listen to your opinions carefully and make improvements.
```
Comment (dingjiaweiww) [inline .github/ISSUE_TEMPLATE/---document-issue-.md]: Thanks for opening a document issue.

File: .github/ISSUE_TEMPLATE/---document-issue-.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+In order to quickly solve your problem, when creating an issue, please provide the following information:
+**Document Information**
+- Title：Please include the keyword "XXX document issue", such as "add document issue" or "paddle.add document issue"
+- Doc Version：Please provide the version of the document, such as develop, 1.8, 2.0RC;
+
+### Document problem description：
```
Comment (dingjiaweiww) [inline .github/ISSUE_TEMPLATE/---document-issue-.md]: Describe the problem

File: .github/ISSUE_TEMPLATE/---document-issue-.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+#### Documentat description is clear?
+For example: I don’t understand this document, I don’t know how to use this API; the document formula is wrong;
```
Comment (dingjiaweiww) [inline .github/ISSUE_TEMPLATE/---document-issue-.md]: The formula in this doc is unclear

Comment (dingjiaweiww) [reply .github/ISSUE_TEMPLATE/---document-issue-.md]: Documentat 这个词打错了

File: .github/ISSUE_TEMPLATE/---document-issue-.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+
+#### Parameter description is clear?
+For example: the function and usage scenarios of this parameter are not explained clearly; the parameter description is incorrect;
```
Comment (dingjiaweiww) [inline .github/ISSUE_TEMPLATE/---document-issue-.md]: The parameters are confusing, including usage, scenarios, default values, etc.

Comment (dingjiaweiww) [inline .github/ISSUE_TEMPLATE/---document-issue-.md]: 改为：The parameters are confusing, including usage, scenarios, default values, etc.

Comment (dingjiaweiww) [inline .github/ISSUE_TEMPLATE/---document-issue-.md]: 改为：The parameters are confusing, including usage, scenarios, default values, etc.

Comment (dingjiaweiww) [inline .github/ISSUE_TEMPLATE/---document-issue-.md]: 改为：The parameters are confusing, including usage, scenarios, default values, etc.

Comment (dingjiaweiww) [inline .github/ISSUE_TEMPLATE/---document-issue-.md]: 改为：The parameters are confusing, including usage, scenarios, default values, etc.

Comment (dingjiaweiww) [inline .github/ISSUE_TEMPLATE/---document-issue-.md]: 改为：The parameters are confusing, including usage, scenarios, default values, etc.

File: .github/ISSUE_TEMPLATE/---document-issue-.md [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+
+#### Return/Shape description is clear
+For example: return/shape description error; shape description is not clear;
```
Comment (dingjiaweiww) [inline .github/ISSUE_TEMPLATE/---document-issue-.md]: Data returned this doc is error, shape returned is not clear.

File: .github/ISSUE_TEMPLATE/---document-issue-.md [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+
+#### Sample code is clear?
```
Comment (dingjiaweiww) [inline .github/ISSUE_TEMPLATE/---document-issue-.md]: The sample code

File: .github/ISSUE_TEMPLATE/---document-issue-.md [hunk 11]
```diff
@@ -0,0 +1,65 @@
...
+For example: there is no sample code; the sample code is not instructable; the sample code does not work; the sample code format is problematic; the sample code has no comments;
```
Comment (dingjiaweiww) [inline .github/ISSUE_TEMPLATE/---document-issue-.md]: For example: no sample code; The sample code is not helpful; The sample code not run well; Format of the sample is not reasonable; The sample code has no comments.

File: .github/ISSUE_TEMPLATE/---document-issue-.md [hunk 12]
```diff
@@ -0,0 +1,65 @@
...
+
+#### Chinese content and English content is consistent?
+For example: Chinese and English API descriptions are inconsistent; Chinese and English API parameters are inconsistent;
```
Comment (dingjiaweiww) [inline .github/ISSUE_TEMPLATE/---document-issue-.md]: Chinese API in this doc is inconsistent with English API, including params, description, sample code, formula, etc.

Comment (dingjiaweiww) [inline .github/ISSUE_TEMPLATE/---document-issue-.md]: Chinese API in this doc is inconsistent with English API, including params, description, sample code, formula, etc.

File: .github/ISSUE_TEMPLATE/---document-issue-.md [hunk 13]
```diff
@@ -0,0 +1,65 @@
...
+
+#### Other
+For example: the document page cannot be opened; the document is missing; there is a dead link in document;
```
Comment (dingjiaweiww) [inline .github/ISSUE_TEMPLATE/---document-issue-.md]: The doc link is broken; The doc page is missing; Dead link in docs.

Comment (dingjiaweiww) [inline .github/ISSUE_TEMPLATE/---document-issue-.md]: The doc link is broken; The doc page is missing; Dead link in docs.

File: .github/ISSUE_TEMPLATE/---document-issue-.md (hunk 1 above)
Comment (TCChenlong) [reply .github/ISSUE_TEMPLATE/---document-issue-.md]: done, thx

File: .github/ISSUE_TEMPLATE/---document-issue-.md (hunk 2 above)
Comment (TCChenlong) [reply .github/ISSUE_TEMPLATE/---document-issue-.md]: done thx

File: .github/ISSUE_TEMPLATE/---document-issue-.md (hunk 3 above)
Comment (TCChenlong) [reply .github/ISSUE_TEMPLATE/---document-issue-.md]: done thx

File: .github/ISSUE_TEMPLATE/---document-issue-.md (hunk 4 above)
Comment (TCChenlong) [reply .github/ISSUE_TEMPLATE/---document-issue-.md]: done thx

File: .github/ISSUE_TEMPLATE/---document-issue-.md (hunk 5 above)
Comment (TCChenlong) [reply .github/ISSUE_TEMPLATE/---document-issue-.md]: done thx

File: .github/ISSUE_TEMPLATE/---document-issue-.md (hunk 6 above)
Comment (TCChenlong) [reply .github/ISSUE_TEMPLATE/---document-issue-.md]: done thx

File: .github/ISSUE_TEMPLATE/---document-issue-.md (hunk 8 above)
Comment (TCChenlong) [reply .github/ISSUE_TEMPLATE/---document-issue-.md]: done thx

File: .github/ISSUE_TEMPLATE/---document-issue-.md (hunk 7 above)
Comment (TCChenlong) [reply .github/ISSUE_TEMPLATE/---document-issue-.md]: done thx

File: .github/ISSUE_TEMPLATE/---document-issue-.md (hunk 8 above)
Comment (TCChenlong) [reply .github/ISSUE_TEMPLATE/---document-issue-.md]: done

File: .github/ISSUE_TEMPLATE/---document-issue-.md (hunk 13 above)
Comment (TCChenlong) [reply .github/ISSUE_TEMPLATE/---document-issue-.md]: done thx

File: .github/ISSUE_TEMPLATE/---document-issue-.md (hunk 12 above)
Comment (TCChenlong) [reply .github/ISSUE_TEMPLATE/---document-issue-.md]: done thx

File: .github/ISSUE_TEMPLATE/---document-issue-.md (hunk 10 above)
Comment (TCChenlong) [reply .github/ISSUE_TEMPLATE/---document-issue-.md]: done thx

File: .github/ISSUE_TEMPLATE/---document-issue-.md (hunk 11 above)
Comment (TCChenlong) [reply .github/ISSUE_TEMPLATE/---document-issue-.md]: done thx

File: .github/ISSUE_TEMPLATE/---document-issue-.md (hunk 9 above)
Comment (TCChenlong) [reply .github/ISSUE_TEMPLATE/---document-issue-.md]: done thx

**Review by jzhang533 — APPROVED** [review]
lgtm

**Pull request closed without merging**